src/block/map/frontend-map.js

```diff
@@ -26,7 +26,7 @@ class StackableMap {
 		return new Promise( resolve => {
 			const script = document.createElement( 'script' )
 			script.id = 'stackable-google-map'
-			script.src = `https://maps.googleapis.com/maps/api/js?key=${ apiKey }&libraries=places`
+			script.src = `https://maps.googleapis.com/maps/api/js?key=${ apiKey }&libraries=places&callback=Function.prototype`
```

Comment:
```suggestion
			// Add callback to prevent warnings.
			script.src = `https://maps.googleapis.com/maps/api/js?key=${ apiKey }&libraries=places&callback=Function.prototype`
```